docs/dev-notes/2026-02-21/refactor-for-workbook/plan.md

```
@@ -0,0 +1,65 @@
+
+✅ **完了**: 29ファイルを移動し、21+ファイルの import パスを更新
+
+### コミット
+
+- `db8c0ff1` - refactor: Move files to features/workbooks (#3193)
+- `1e263a92` - refactor: Move and reconfigure workbooks to features (#3193)
+- `e9135703` - docs: Remove completed workbooks from feature split plan (#3193)
+
+### 完成したディレクトリ構造
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)